plugins/obs-ffmpeg/obs-ffmpeg-vaapi.c

```diff
@@ -700,7 +860,7 @@ static obs_properties_t *vaapi_properties(void *unused)
 				char card[128];
 				int ret = snprintf(card, sizeof(card),
 						   "Card%d: %s", i - 28, path);
-				if (ret >= sizeof(card))
+				if (ret >= (int)sizeof(card))
```

Comment:
Except this change that is already part of #6759, The vaapi-utils code LGTM.

Reply:
Looks like the same fix, so it should be fine.

Reply:
And it was already merged in the another PR. Resolving.